models/login_source.go

```diff
@@ -28,25 +28,25 @@ type LoginType int
 
 // Note: new type must append to the end of list to maintain compatibility.
 const (
-	LOGIN_NOTYPE LoginType = iota
-	LOGIN_PLAIN            // 1
-	LOGIN_LDAP             // 2
-	LOGIN_SMTP             // 3
-	LOGIN_PAM              // 4
-	LOGIN_DLDAP            // 5
+	LoginNotype LoginType = iota
+	LoginPlain            // 1
+	LoginLdap             // 2
```

Comment:
LDAP, SMTP, PAM, DLDAP are all abbreviations and should be upper-case (Golang allows this)